BUILDING

```diff
@@ -43,7 +43,7 @@ Platform-specific notes
   pass the relevant directories directly to ./configure:
       ./configure CPPFLAGS="-I/usr/local/opt/openssl/include" LDFLAGS="-L/usr/local/opt/openssl/lib"
 
-  Alternatively, you may with to add these lines to your $HOME/.profile file:
+  Alternatively, you may wish to add these lines to your $HOME/.profile file:
```

Comment:
This PR is pretty trivial, and normally I'd prefer to wait until there's more important changes.

But that "with -> wish" typo was horrible (a textual lisp???), and I've sat on this for 5 or 6 weeks, so I figured it's worth getting it fixed.